claudia-lab/test/read-file-helper-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const readfilehelper = require('../lib/read-file-helper.js');
+const assert = require('assert');
+
+describe('testing readfilehelper()', function(){
+  it('should return correct string', function(done){ //tells it that it's done with asychronous tests
+    readfilehelper(function(hexArray){
+      console.log(hexArray);
```

Comment:
@loomnugget be sure to remove any console.logs prior to pushing code to master - if needed, you can create a branch that includes these additions (ie: features/my_feature_testing)